tests/sparse_map_tests.cpp

```diff
@@ -1132,6 +1132,11 @@ BOOST_AUTO_TEST_CASE(
   BOOST_CHECK_EQUAL(map.bucket_count(), 64);
   BOOST_CHECK_EQUAL(map.max_load_factor(), 0.8f);
 
+  /**
+   * Current map load threshold for rehashing is expected to be 52 elements.
```

Comment:
The load rehash threshold is 51 not 52. I think though that the comment can eventually be removed.